drivers/watchdog/Kconfig.opentitan

```diff
@@ -0,0 +1,15 @@
+# OpenTitan Always-On Timer support
+
+# Copyright (c) 2023, Rivos Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+config WDT_OPENTITAN
+	bool "OpenTitan Always-On (AON) Timer"
+	depends on SOC_RISCV_OPENTITAN
+	depends on DT_HAS_LOWRISC_OPENTITAN_AONTIMER_ENABLED
+	default y
+	select HAS_WDT_MULTISTAGE
+	select WDT_MULTISTAGE
```

Comment:
why select both? This looks redundant to me

Reply:
`WDT_MULTISTAGE` depends on `HAS_WDT_MULTISTAGE`. It doesn't allow me to select `WDT_MULTISTAGE` directly without selecting the former, and the multistage feature uses `CONFIG_WDT_MULTISTAGE`. I'm not sure if there's a more appropriate way to handle this

Reply:
It seems `HAS_WDT_MULTISTAGE` isn't used anywhere, and I'm not sure what the original intent/purpose of it is. If it is safe to remove I will add a patch to do so, to be included in this series.

Reply:
In this case, I am writing a driver that depends on the `WDT_MULTISTAGE` feature, so it is necessary to select both.

Reply:
Can I resolve this? After a bit of thinking I do have an idea of turning the multi-stage component optional, if you'd like: it would involve skipping the interrupt step and going straight into the reset.

Reply:
if you think you can resolve this go ahead. Just mark this PR DNM until this is ready again.

Reply:
Isn't this a case where driver _can_ support WDT_MULTISTAGE? Because that means driver should just `select HAS_WDT_MULTISTAGE` and support the feature by adding code conditional to `WDT_MULTISTAGE`, and then, clients will select `WDT_MULTISTAGE` if they need it. Am I missing something? @martinjaeger

Reply:
Alright, I changed the feature.